sdcm/cluster.py

```diff
@@ -2555,21 +2556,24 @@ def restart_node_with_resharding(self, murmur3_partitioner_ignore_msb_bits: int
         resharding_started = list(search_reshard_start)
         resharding_finished = list(search_reshard_finish)
 
-        if resharding_started:
-            self.log.debug(f'Resharding has been started successfully '
-                           f'(murmur3_partitioner_ignore_msb_bits={murmur3_partitioner_ignore_msb_bits})')
-        else:
-            raise Exception(f'Resharding has not been started '
-                            f'(murmur3_partitioner_ignore_msb_bits={murmur3_partitioner_ignore_msb_bits}) '
-                            'Check the log for the details')
+        # If tablets in use, skipping resharding validation since it doesn't work the same as vnodes
+        with self.parent_cluster.cql_connection_patient(self) as session:
+            if not is_tablets_feature_enabled(session=session):
```

Comment:
please add the SkipPerIssue, so it would point to the relevant issue.

also for docs, and also so it would start working when resolved

Reply:
@fruch fixed.